packages/docs/page-config/styles/css-variables/examples/profile.vue

```diff
@@ -25,10 +25,12 @@
         </div>
 
         <va-card-actions class="flex pt-2">
+          <!-- class -ml-2 and -ml-[0.5rem] makes the Follow button move left a little bit
+          how should I specify in class? -->
           <va-button
             preset="secondary"
             color="#fff"
-            style="margin-left: -0.5rem;"
+            class="-ml-[0.5rem]"
```

Comment:
Setting class to `-ml-2` and `-ml-[0.5rem]` cause the follow button to move left a little bit, why is that?